secrets/secrets-kubernetes/deploy/dev/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../impersonation/
+
+images:
+  - name: wasmcloud-secrets
+    newName: ghcr.io/wasmcloud/wasmcloud-contrib/secrets-kubernetes
+    newTag: dev
+
+patches:
+  - patch: |-
+      - op: replace
+        path: "/spec/template/spec/containers/0/args"
+        value:
+          - "--backend-seed=SXAD2NAUWO6YNEFMY4FQT7D45VLLWFOZDVHCENMPHCWA6ABBLZ4OBBKGKQ"
```

Comment:
Just to be sure, is this file an example, or something that is used w/ the backend?

Reply:
it's for local development only, so developers don't need to fiddle with keys to stand up a test backend